README.md

```diff
@@ -279,8 +279,13 @@ change to adapt it to your specific needs.
     The maximum time in milliseconds the crawler will wait for async listeners.
 * `crawler.userAgent="Node/simplecrawler <version> (https://github.com/cgiffard/node-simplecrawler)"` -
     The user agent the crawler will report.
+* `crawler.decompressResponses=true` -
```

Comment:
Good point, I think it'd be pretty sound to automatically decode responses. Could either change the default for 1.0.0 or just go ahead and remove the option. I'm not sure what happens to binary resources today, though. Might be worth looking into to make sure it's done logically.